app/assets/javascripts/intl-tel-input.js.erb

```diff
@@ -0,0 +1,22 @@
+// Setting hidden input for intl code
+
+var countryData = $.fn.intlTelInput.getCountryData(),
+  telInput = $("#user_phone_form_phone");
+
+
+// initialise plugin
+telInput.intlTelInput({
+  utilsScript: "<%= asset_path 'intl-tel-number/utils.js' %>",
+  preferredCountries: ["us", "ca"]
+
+});
```

Comment:
Is this the only way to load the `utils.js` script? Would it be possible to use the `import` method instead, like in our other JS files? This would allow us to make this a plain JS file instead of ERB.

If ERB is the only way to do this, then we would need to add the following to `config/webpack/environment.js`:
```js
const erb = require('.loaders/erb')
environment.loaders.append('erb', erb)
```
Those 2 lines would go in between the 2 lines that are in the file currently.

We also would need to add a file called `config/webpack/loaders/erb.js` with the following in it:
```js
module.exports = {
  test: /\.erb$/,
  enforce: 'pre',
  exclude: /node_modules/,
  use: [{
    loader: 'rails-erb-loader',
    options: {
      runner: 'bin/rails runner'
    }
  }]
}
```

and then add `"rails-erb-loader": "^5.2.1"` to our `dependencies` in `package.json`.

And finally add ` - .erb` to the `extensions` list in `config/webpacker.yml`.